src/main/java/graphql/language/AstPrinter.java

```diff
@@ -1,9 +1,11 @@
 package graphql.language;
 
+import com.google.common.io.CharStreams;
```

Comment:
Note: we only shade in selected classes of Guava, before merging in this PR the Gradle config should be changed to shade in this new class

Reply:
Our copy the code in - 

Hmmm com.google.common.io.AppendableWriter is kinda large.

But we dont really want the whole`com.google.common.io` classes in here - just some select ones say


Can we tighten this down some how in shadow jar config ??

Reply:
```
 relocate('com.google.common', 'graphql.com.google.common') {
        include 'com.google.common.collect.*'
        include 'com.google.common.base.*'
        include 'com.google.common.math.*'
        include 'com.google.common.primitives.*'
    }
```

is what we have today - I wonder if we can do specific classes say rather than all of `common.io`

Reply:
Thanks @kilink for updating the PR to use a non-Guava alternative, that's a great idea